get-deps

```diff
@@ -114,46 +146,105 @@ if test -e /etc/arch-release ; then
     'xcb-util-keysyms' \
     'xcb-util-wm'
   exit $?
+}
+
+bsd_deps() {
+  PKG="$SUDO pkg"
+  $PKG install -y \
+    'cmake' \
+    'curl' \
+    'egl-wayland' \
+    'expat' \
+    'fontconfig' \
+    'gcc' \
+    'gettext' \
+    'git' \
+    'gmake' \
+    'libxcb' \
+    'libxkbcommon' \
+    'mesa-devel' \
+    'openssl' \
+    'p5-ExtUtils-MakeMaker' \
+    'perl5' \
+    'pkgconf' \
+    'python3' \
+    'rust' \
+    'wayland' \
+    'libxcb' \
+    'xcb-util' \
+    'xcb-util-image' \
+    'xcb-util-keysyms' \
+    'xcb-util-wm' \
+    'z' \
+    'zip'
+  exit $?
+}
+
+if test -e /etc/os-release; then
+  . /etc/os-release
+
+  case $ID in
+    centos|fedora|rhel)
+      fedora_deps
```

Comment:
I'd prefer to have the decision to `exit` be moved out of the various `_deps` functions and placed here, so that the control flow is more clear.

```suggestion
      fedora_deps
      exit $?
```

if for some reason that isn't workable, then I would advocate for renaming those functions to be something like `install_fedora_deps_and_exit` to make it clearer that they don't return.

Reply:
Sure 👍